[2025-02-05T06:41:16.429+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-05T06:41:16.480+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: task_group_example.reporting.send_report scheduled__2024-02-28T00:00:00+00:00 [queued]>
[2025-02-05T06:41:16.483+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: task_group_example.reporting.send_report scheduled__2024-02-28T00:00:00+00:00 [queued]>
[2025-02-05T06:41:16.483+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2025-02-05T06:41:16.487+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): reporting.send_report> on 2024-02-28 00:00:00+00:00
[2025-02-05T06:41:16.489+0000] {standard_task_runner.py:57} INFO - Started process 2918 to run task
[2025-02-05T06:41:16.490+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'task_group_example', 'reporting.send_report', 'scheduled__2024-02-28T00:00:00+00:00', '--job-id', '313', '--raw', '--subdir', 'DAGS_FOLDER/task_group_dag.py', '--cfg-path', '/tmp/tmpya5cycxj']
[2025-02-05T06:41:16.491+0000] {standard_task_runner.py:85} INFO - Job 313: Subtask reporting.send_report
[2025-02-05T06:41:16.497+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-05T06:41:16.509+0000] {task_command.py:415} INFO - Running <TaskInstance: task_group_example.reporting.send_report scheduled__2024-02-28T00:00:00+00:00 [running]> on host 3849bfb82152
[2025-02-05T06:41:16.513+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-05T06:41:16.537+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='task_group_example' AIRFLOW_CTX_TASK_ID='reporting.send_report' AIRFLOW_CTX_EXECUTION_DATE='2024-02-28T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-28T00:00:00+00:00'
[2025-02-05T06:41:16.537+0000] {logging_mixin.py:151} INFO - 执行任务: 发送报告
[2025-02-05T06:41:16.538+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-02-05T06:41:16.540+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=task_group_example, task_id=reporting.send_report, execution_date=20240228T000000, start_date=20250205T064116, end_date=20250205T064116
[2025-02-05T06:41:16.548+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-02-05T06:41:16.555+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
